packages/auth/amplify_auth_cognito_dart/lib/src/flows/hosted_ui/hosted_ui_platform_html.dart

```diff
@@ -13,27 +13,31 @@ class HostedUiPlatformImpl extends HostedUiPlatform {
   HostedUiPlatformImpl(super.dependencyManager) : super.protected();
 
   /// The base URL
```

Comment:
It's not required any longer and I agree this wasn't very helpful to begin with. But in general, doc comments on private fields can still be helpful since they appear when you hover over them in code.